src/examples/forms/checkbox/Hint.tsx

```diff
@@ -14,20 +14,24 @@ const StyledField = styled(Field)`
   margin-top: ${p => p.theme.space.xs};
 `;
 
+const StyledHint = styled(Hint)`
+  ${p => `margin-${p.theme.rtl ? 'right' : 'left'}: ${p.theme.space.base * 6}px;`}
+`;
```

Comment:
If this placement of Hint with respect to each of these fields is our opinion (and not just an example), then I wonder if these styles actually belong in the [component](https://github.com/zendeskgarden/react-components/blob/main/packages/forms/src/styled/checkbox/StyledCheckHint.ts) itself.

Reply:
oy, you're totally right. And they are accounted for when we stick with the structure as-is 🤦